### PR TITLE
Make package more ros pythonic

### PR DESCRIPTION
**Pull request by garyservin**
A bit of cleanup and some updates to make the package a bit more ROS-compliant. I think  (and may be wrong) we should separate more the logic from the node and create a RoDIRobot or similar class that takes care of the logic side.

### CODE REVIEW

**Comment by garyservin** [comment]
~~Do not merge yet!!~~ Ready to merge.

File: src/rodi_robot/main.py [hunk 1]
```diff
@@ -5,12 +5,17 @@
 from sensor_msgs.msg import Range
 from rodi_robot.transport import Transport
 
+import threading
+
 
 class RodiRobot(object):
 
+    ''' Robot class to add ROS support to RoDI'''
+
```
Comment (martinezjavier) [inline src/rodi_robot/main.py:12]: I think this comment should be in the previous patch that includes this class.

File: src/rodi_robot/main.py [hunk 2]
```diff
@@ -30,7 +35,28 @@ def __init__(self):
         self.us_sensor.min_range = 0.2
         self.us_sensor.max_range = 4.0
 
+        if self.timeout > 0:
+            self._timer = threading.Timer(self.timeout, self._timeout)
+            self._set_timeout()
```
Comment (martinezjavier) [inline src/rodi_robot/main.py]: Do we really need this timer? I was thinking on a simpler patch where the last timestamp could be stored when a message was received (i.e: rospy.Time.now() ) and check for the delta on the rospy main loop.

**Comment by garyservin** [comment]
I've addressed the comments and updated the PR :grin:

**Comment by martinezjavier** [comment]
It looks good to me now, thanks a lot for addressing the issues I pointed out!

**Pull request merged**